### PR TITLE
Editorial: Add landmark language to definitions

### LINKED ISSUE #84: WAI-ARIA 1.1: Editorial issue re. definition for landmark roles
Filed at https://lists.w3.org/Archives/Public/public-pfwg-comments/2015JulSep/0008.html by Sailesh Panchang spanchang02@yahoo.com 

The definitions of form and search roles identify them as landmark roles. The definitions for other landmark roles do not  indicate this. Maybe they should . This is so in Aria 1.0 and 1.1. 
Admittedly section 5.3.4 does list them all.  
Thanks,
Sailesh Panchang

### PR DESCRIPTION
**Pull request by jnurthen**
resolves #84

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1060.html" title="Last updated on Oct 2, 2019, 5:40 PM UTC (7351c11)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1060/2b5a43e...7351c11.html" title="Last updated on Oct 2, 2019, 5:40 PM UTC (7351c11)">Diff</a>

### CODE REVIEW

**Comment by carmacleod** [comment]
@jnurthen Your changes look good.

Wondering, though, why `contentinfo` is "A **_large_** **_perceivable_** landmark" when in fact it is no larger or more perceivable than a `banner` or any other landmark. Can we delete the 2 words "large perceivable"?

Similarly, for `region`: "A **_perceivable_** landmark", can we delete the word "perceivable"?

**Comment by jnurthen** [comment]
@carmacleod happy to make those changes. I was simply making the minimum changes possible to resolve the comment.

**Comment by carmacleod** [comment]
@jnurthen Understood. It just struck me as "off" when reviewing.  :)

**Comment by carmacleod** [comment]
Thanks!

File: index.html [hunk 1]
```diff
@@ -1386,7 +1386,7 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="banner">
 			<rdef>banner</rdef>
 			<div class="role-description">
-				<p>A region that contains mostly site-oriented content, rather than page-specific content.</p>
+				<p>A <rref>landmark</rref> region that contains mostly site-oriented content, rather than page-specific content.</p>
```
Comment (joanmarie) [inline index.html]: Since region is a role, perhaps we should prefer "A landmark that contains...."

File: index.html [hunk 2]
```diff
@@ -2306,7 +2306,7 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="complementary">
 			<rdef>complementary</rdef>
 			<div class="role-description">
-				<p>A supporting section of the document, designed to be complementary to the main content at a similar level in the DOM hierarchy, but remains meaningful when separated from the main content.</p>
+				<p>A <rref>landmark</rref> region, designed to be complementary to the main content at a similar level in the DOM hierarchy, but remaining meaningful when separated from the main content.</p>
```
Comment (joanmarie) [inline index.html]: Again, should we remove "region"?

File: index.html [hunk 3]
```diff
@@ -2456,7 +2456,7 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="contentinfo">
 			<rdef>contentinfo</rdef>
 			<div class="role-description">
-				<p>A large perceivable region that contains information about the parent document.</p>
+				<p>A <rref>landmark</rref> region that contains information about the parent document.</p>
```
Comment (joanmarie) [inline index.html]: Ditto.

File: index.html [hunk 4]
```diff
@@ -4715,7 +4715,7 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="main">
 			<rdef>main</rdef>
 			<div class="role-description">
-				<p>The main content of a document.</p>
+				<p>A <rref>landmark</rref> region containing the main content of a document.</p>
```
Comment (joanmarie) [inline index.html]: Ditto.

File: index.html [hunk 5]
```diff
@@ -5589,7 +5589,7 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 		<div class="role" id="navigation">
 			<rdef>navigation</rdef>
 			<div class="role-description">
-				<p>A collection of navigational <a>elements</a> (usually links) for navigating the document or related documents.</p>
+				<p>A <rref>landmark</rref> region containing a collection of navigational <a>elements</a> (usually links) for navigating the document or related documents.</p>
```
Comment (joanmarie) [inline index.html]: Ditto.

File: index.html [hunk 6]
```diff
@@ -6546,7 +6546,7 @@ <h5>Presentational Roles Conflict Resolution</h5>
 		<div class="role" id="region">
 			<rdef>region</rdef>
 			<div class="role-description">
-				<p>A perceivable <rref>section</rref> containing content that is relevant to a specific, author-specified  purpose and sufficiently important that users will likely want to be able to navigate to the section easily and to have it listed in a summary of the page. Such a page summary could be generated dynamically by a user agent or assistive technology.</p>
+				<p>A <rref>landmark</rref> containing content that is relevant to a specific, author-specified  purpose and sufficiently important that users will likely want to be able to navigate to the section easily and to have it listed in a summary of the page. Such a page summary could be generated dynamically by a user agent or assistive technology.</p>
```
Comment (joanmarie) [inline index.html]: This one I like. :smile:

**Pull request merged**